backend/de.metas.business/src/main/sql/postgresql/ddl/functions/Reset_System_Base_Language.sql

```diff
@@ -0,0 +1,82 @@
+CREATE OR REPLACE FUNCTION Reset_System_Base_Language(p_ad_language_id numeric = -1) RETURNS VOID
```

Comment:
I would rename the function as `setBaseLanguage` because that's what it actually does.